tasks/pkg-windows.yml

```diff
@@ -4,6 +4,16 @@
     msg: "The Datadog ansible role does not currently support Agent 5"
   when: datadog_agent5
 
+- name: Download windows datadog agent
```

Comment:
not the right name here, probably bad copy-pasta :)

Reply:
:+1: